Classes/Domain/Model/Dto/CommentItem.php

```
@@ -66,8 +66,47 @@ public function getEditUri(): string
+    {
+        if (!$this->isResolved()) {
+            return '';
+        }
+
+        $resolvedData = json_decode($this->data['resolved'], true);
+        if (!is_array($resolvedData) || !isset($resolvedData['date'])) {
+            return '';
+        }
+
+        $timestamp = (int)$resolvedData['date'];
+        return $timestamp;
```

PHPStan (2.0.3)

Method Xima\XimaTypo3ContentPlanner\Domain\Model\Dto\CommentItem::getResolvedDate() should return string but returns int.


(return.type)